src/notificationhandler.h

```diff
@@ -57,8 +63,9 @@ class NotificationHandler : public QObject {
                       const QString& message, int timerMsec) = 0;
 
  private:
-  virtual void notifyInternal(Message type, const QString& title,
-                              const QString& message, int timerMsec);
+  virtual void notifyInternal(
+      Message type, const QString& title, const QString& message,
+      int timerMsec = Constants::DEFAULT_NOTIFICATION_TIMER);
```

Comment:
can we keep timerMSec as a "fix" param?

Reply:
Hm, I am not sure I know what you mean. Would you mind explaining what you mean by "fix param"?